packages/opentelemetry-core/src/context/propagation/B3Propagator.ts

```diff
@@ -93,7 +116,10 @@ export class B3Propagator implements HttpTextPropagator {
         traceId,
         spanId,
         isRemote: true,
-        traceFlags: isNaN(Number(options)) ? TraceFlags.NONE : Number(options),
+        // Set traceFlags as 1 if debug is 1
```

Comment:
Turned traceFlags to 1 when debug is set to 1 so that the behavior is the same when as having traceFlags is 1 since debug is like setting traceFlags to 1